_rules/aria-state-or-property-not-prohibited-yyw7k3.md

```diff
@@ -0,0 +1,150 @@
+---
+id: yyw7k3
+name: ARIA state or property is not prohibited
+rule_type: atomic
+description: |
+  This rule checks that WAI-ARIA states or properties are not prohibited for the element they are specified on.
+accessibility_requirements:
+  wcag-technique:ARIA5: # Using WAI-ARIA state and property attributes to expose the state of a user interface component
+    forConformance: false
+    failed: not satisfied
+    passed: further testing needed
+    inapplicable: further testing needed
+  aria12:state_property_processing:
+    title: ARIA 1.2, 8.6 State and Property Attribute Processing
+    forConformance: true
+    failed: not satisfied
+    passed: satisfied
+    inapplicable: satisfied
+  wcag20:1.3.1: # Info and Relationships (A)
+    secondary: true
+  wcag20:4.1.2: # Name, Role, Value (A)
+    secondary: true
+input_aspects:
+  - Accessibility Tree
+  - CSS styling
+  - DOM Tree
+acknowledgments:
+  authors:
+    - Trevor Bostic
+  previous_authors:
+    - Anne Thyme Nørregaard
+    - Jean-Yves Moyen
+  funding:
+    - WAI-Tools
+  assets:
+    - JFK's "We Choose the Moon" speech excerpt is courtesy of NASA.
+---
+
+## Applicability
+
+This rule applies to any [WAI-ARIA state or property][] that is specified on an [HTML or SVG element][namespaced element] that is [included in the accessibility tree][].
+
+## Expectation
+
+No test target (i.e., state or property) is prohibited on the semantic role of the element on which it is specified.
```

Comment:
I'm not a fan of the "(i.e. state or property)" bit. If we were to do this in all rules I'm concerned this would get messy. If we feel that "test target" as a term is confusing we should have a separate conversation about that, and then adjust all rules to fit the outcome. I don't think we want to introduce changes like this without thinking through the broader implications.